modules/azure/aks-global/aad-group.tf

```diff
@@ -1,35 +1,35 @@
 resource "azuread_group" "view" {
   for_each     = { for ns in var.namespaces : ns.name => ns }
-  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${each.value.name}${var.group_name_separator}view"
+  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${var.location_short}${var.group_name_separator}${each.value.name}${var.group_name_separator}view"
   #description             = "Members of this group will have view access to the ${each.value.name} namespace."
   prevent_duplicate_names = true
   security_enabled        = true
 }
 
 resource "azuread_group" "edit" {
   for_each     = { for ns in var.namespaces : ns.name => ns }
-  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${each.value.name}${var.group_name_separator}edit"
+  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${var.location_short}${var.group_name_separator}${each.value.name}${var.group_name_separator}edit"
   #description             = "Members of this group will have edit access to the ${each.value.name} namespace."
   prevent_duplicate_names = true
   security_enabled        = true
 }
 
 resource "azuread_group" "cluster_admin" {
-  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}clusteradmin"
+  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${var.location_short}${var.group_name_separator}clusteradmin"
```

Comment:
As pointed out during our face 2 face review aks-global isn't really global.
Would it be reasonable to create an actual aks-global where we would store a global admin group that could be a part of this group.

That way cluster admins only need to be a part of a single group to get cluster-admin on all the aks clusters.
This might also be a possible good solution for azad-kube-proxy config.